webapi/tct-widget02-w3c-tests/scripts/w3c_widget_installfail.sh

```diff
@@ -41,18 +41,16 @@ p_name=$1
 APP_NAME=${p_name%%.wgt}
 uninstall_app $APP_NAME
 find_app $APP_NAME
-if [ ! -z "$pkgids"  ]
-  then
-  echo -e  "Fail to uninstall the existed widget!"
+if [ ! -z "$pkgids" ];then
```

Comment:
`[[ ... ]]` is preferred over `[ ... ]`.

See https://google-styleguide.googlecode.com/svn/trunk/shell.xml?showone=Test,_[_and_[[#Test,_[_and_[[